code-challenges.test.js

```diff
@@ -41,17 +62,37 @@ var secretCodeWord3 = "Eccentric"
 var arrayOfWords1 = ["Apple", "Banana", "Plum", "Cherry", "Kiwi"]
 // Expected output: ["Apple", "Banana"]
 var arrayOfWords2 = ["Mango", "Orange", "Apricot", "Blueberry", "Peach"]
-// Expected output: ["Mango", "Banana", "Peach"]
-
-
-
-
-
-// b) Create the function that makes the test pass.
-
-
-
-
+// Expected output: ["Mango", "Banana", "Peach","Orange"]
+
+Describe("returnLetter",() =>{
+    test("takes in a an array and returns all the words that contain letter a",() =>{
+        let arayOfWords1 = ["Apple", "Banana", "Plum", "Cherry", "Kiwi"]
+        let arrayOfWords2 = ["Mango", "Orange", "Apricot", "Blueberry", "Peach"]
+        expect(returnLetter(secretCodeWord1).toEqual(["Apple", "Banana"]))
+        expect(returnLetter(secretCodeWord2).toEqual(["Mango","Banana", "Peach","Orange"]))
+   
+
+
+// declare a function that takes in an array as an argument
+// create a new, emoty arry to store all words that contain the letter "a"
+// create a loop that goes through ant array i pass to the function
+// use a conitional statement to check if that word contains the letter "a"
+// if that words contains the letter "a", I want to push the expected output into new arr.
+// return the new original blank array
+// pass my test 
+const returnLetter = (arr) => {
+    let arrayOfAs = []
+    for(let i= 0; i < arr.length; i++){
+        if(arr[i].charAt(0) === "A"){
+            arrayOfAs.push('arrayofAs')
+        }  
+        }
+        return arrayOfAs
```

Comment:
so here you're trying to "return" and empty array. your return should change or mutate your empty array